projects/documentation/src/components/layout.ts

```diff
@@ -212,6 +221,10 @@ export class LayoutElement extends LitElement {
         this.open = !this.open;
     }
 
+    get systemValue() {
+        return this.system;
+    }
```

Comment:
Can we use `this.system` where `this.systemValue` is currently leveraged to simplify?